tests/reframe/config/settings_cartesius.py

```diff
@@ -0,0 +1,119 @@
+site_configuration = {
```

Comment:
do we want deployment specific files in here?

Reply:
No, sorry, didn't realize this was still in there :) I'll take it out.